src/app/phast/losses/flue-gas-losses/flue-gas-losses-form-mass/flue-gas-losses-form-mass.component.html

```diff
@@ -116,9 +116,9 @@
     <div class="form-group">
       <label class="small" for="{{'moistureInAirComposition'+idString}}">Moisture in Combustion Air</label>
       <div class="input-group" [ngClass]="{'indicate-different': compareMassMoistureInAirComposition(), 'error': warnings.moistureInAirCompositionWarning !== null, 'invalid': flueGasLossForm.controls.moistureInAirComposition.invalid}">
-        <input [readonly]="!baselineSelected" name="{{'moistureInAirComposition_'+lossIndex}}" type="number" step="any"
+        <input [readonly]="!baselineSelected" name="{{'moistureInAirComposition_'+lossIndex}}" type="number" step="any" 
           min="0" max="100" class="form-control" formControlName="moistureInAirComposition" id="{{'moistureInAirComposition'+idString}}"
-          onfocus="this.select();" (input)="calcExcessAir()" (focus)="focusField('moistureInAirComposition')" >
+          onfocus="this.select();" (input)="calcExcessAir()" (focus)="focusField('moistureInAirComposition')" [ngModel] = "humidityRatio | number:'1.0-2' | removeCommas" (ngModelChange) = "humidityRatio=$event">
```

Comment:
This should just be [(ngModel)]="humidityRatio"

Reply:
If you have a reason for not doing it that way let me know

Reply:
This way allows pipes to be used to format the result from the modal, but doesn't have pipes when storing data manually entered into the input (where their use wouldn't make sense).